faasmcli/faasmcli/tasks/docker_tasks.py

```diff
@@ -66,8 +66,6 @@ def build(ctx, c, nocache=False, push=False):
     faasm_ver = get_faasm_version()
 
     for container in c:
-        # Check if we need to template a special dockerignore file It seems the
-        # dockerignore file needs to be at <dockerfile_path>.dockerignore
```

Comment:
This comment is no longer valid and should have been removed in the PR that removed the custom dockerignore feature.